src/test/java/com/gooddata/notification/NotificationServiceAT.java

```diff
@@ -6,15 +6,48 @@
 package com.gooddata.notification;
 
 import static java.util.Collections.singletonMap;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.core.Is.is;
 
 import com.gooddata.AbstractGoodDataAT;
+import com.gooddata.account.Account;
 import org.testng.annotations.Test;
 
+import java.util.Arrays;
+
 public class NotificationServiceAT extends AbstractGoodDataAT {
 
+    private ChannelConfiguration channel;
+
     @Test(groups = "notification", dependsOnGroups = "project")
     public void triggerProjectEvent() throws Exception {
         gd.getNotificationService().triggerEvent(project, new ProjectEvent("sdk.event.test", singletonMap("test", "map")));
     }
 
+    @Test(groups = "notification", dependsOnGroups = "project")
+    public void createChannel() throws Exception {
+        final Account current = gd.getAccountService().getCurrent();
+
+        channel = gd.getNotificationService().createChannel(current,
```

Comment:
Shouldn't there be sth like `removeChannel` in `@After` method?

Reply:
added